app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -407,10 +407,11 @@ export default {
                 </template>
                 <!-- users found in dropdown -->
                 <template #item="{ props, item }">
+                  {{ item.raw }}
```

Comment:
Was this for debugging? I'm seeing:
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/a051aeac-8f9e-4ada-af28-ef3f1412d559)

Reply:
oh shoot yeah haha, i'll remove that while i'm fixing the previous issue

Reply:
This should be resolved now